### PR TITLE
Added Animations in the Website

### LINKED ISSUE #7: [FIX] Animation in Website
I have seen the preview of the Website. I want to add the animations in the website in different actions. Like toggling any action on button press (Dark mode toggle, toggle comment box, etc). I will add the smooth Animations in the website. 
Please assign this issue to me as I am participant in the hacktoberfest 2024.

### PR DESCRIPTION
**Pull request by Yuvraj960**
FIXED #7 
Added Animation in dark mode toggle. 
Added animation in toggling of comments and description box

### CODE REVIEW

**Comment by Yuvraj960** [comment]
@kunalbhatia2601 Hey Bro! I have added animations and transitions in the website. but there are some wierd spacing issues in the style.css file. Actually, it happened because of my VSCode formatting option which removes the extra spacing automatically in the file.

**Comment by MITARPALGROVER** [comment]
bro cant merge due to conflicts

**Comment by Yuvraj960** [comment]
@MITARPALGROVER Bro! I have resolved some major conflicts. Now one minor conflict is left thats I am unable to resolve due to some issue in the retrieving back the previous commits. So I can tell you the solution and can you please resolve that minor issue on merge. The simple resolve is you have to accept the code until line 314 after that some repititive code is there which is trash one

**Comment by Yuvraj960** [comment]
@MITARPALGROVER  
Bro! I have resolved some major conflicts. Now one minor conflict is left thats I am unable to resolve due to some issue in the retrieving back the previous commits and also the other commits in the Repo contributed to this conflict. Some other contributor has written media queries while I have written the animation code at same place. The simple resolve to this issue to take the combination of both incoming. I am not able to resolve this because at my local repo, there are no conflits according to the Git in VSCode. So can you please do this simple resolving of issue and submit my PR as merged.

**Comment by MITARPALGROVER** [comment]
bro ik dikkat aur hai apne changes ko pull nhi kara hua like 3 days ago changes hue the iss repository me toh apne woh changes pull nhi kare hue 
kya app please woh pull karke usme changes karke ik new pr bhej sakte ha?

**Comment by Yuvraj960** [comment]
@MITARPALGROVER Ok bro, I will do it. But it will take some more time as already the PR Structure is very complex.

**Comment by Yuvraj960** [comment]
@MITARPALGROVER Bro I have pushed 8 commits in order to resolve the conflicts. But still whenever I add animations queries more conflicts are there. Suggest me some way out to resolve the conflicts!

File: style.css [hunk 1]
```diff
@@ -232,3 +232,159 @@ body {
   background-color: #444;
   color: white;
 }
+
```
Comment (Yuvraj960) [inline style.css:235]: Here I have to attach the animation but whenever I add the code the conflicts arise.

**Comment by Yuvraj960** [comment]
@MITARPALGROVER Sorry for so many commits. But the issue is not resolving, so I have to insert and delete the code from one place to another many times in order to fix the conflict. I hope now my PR is merged!

**Comment by MITARPALGROVER** [comment]
image

bro can you please explain me what's this saying coz i am totally new to all this stuff......

**Comment by Yuvraj960** [comment]
@MITARPALGROVER Please ask other repo maintainers for this stuff, as they can explain you better. 
@kunalbhatia2601 Kindly look into this and Merge my PR

**Pull request closed without merging**